README.md

```diff
@@ -148,3 +148,19 @@ curl "http://docker.localhost:5080/katsu/api/datasets" \
 curl "http://docker.localhost:3333/htsget/v1/variants/NA20787" \
      -H 'Authorization: Bearer <access_token>'
 ```
+
+## Ingest clinical data into katsu MoH
+
+Before you can ingest the clinical data, you need to obtain the data from the [clinical_ETL](https://github.com/CanDIG/clinical_ETL_data) and put it in the `data` folder, then set the environment variable `MOH_DATA_LOCATION`:
+
+NOTE: if you just want to use the [synthetic data](https://raw.githubusercontent.com/CanDIG/katsu/sonchau/moh_part_22/chord_metadata_service/mohpackets/data/small_dataset/synthetic_data/), you can skip this step.
```

Comment:
Since posting this I realized that the actual part of the URL that matters is the stuff after it (e.g. `Program.json`). Unsure if this should be kept as a link (`[]()`) in the Markdown though.

Reply:
yea, I can make it a normal link instead of raw